backend/app/services/knowledge/knowledge_service.py

```
@@ -852,6 +875,77 @@ def get_active_document_text_length_stats(
+            return {}
+
+        from sqlalchemy import func
+
+        results = (
+            db.query(
+                KnowledgeDocument.kind_id,
+                func.count(KnowledgeDocument.id).label("count"),
+            )
+            .filter(
+                KnowledgeDocument.kind_id.in_(kb_ids),
+                KnowledgeDocument.is_active == True,
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `KnowledgeDocument.is_active:` for truth checks

Replace with `KnowledgeDocument.is_active`

(E712)